### PR TITLE
Added back button in order view

### PR DESCRIPTION
**Pull request by phillipnunes**
image

### CODE REVIEW

**Comment by vellip** [comment]
Can you make this a separate component? Then we can use it on other forms as well.. and style wise maybe like here https://www.figma.com/file/MzDFDSP7oMdUrIZB7fjpVt/Covexit?node-id=197686%3A4542
Also, for icons we use react-icons. The ones you'll find on Figma are from feather-icons https://feathericons.com/

**Comment by phillipnunes** [comment]
> Can you make this a separate component? Then we can use it on other forms as well.. and style wise maybe like here https://www.figma.com/file/MzDFDSP7oMdUrIZB7fjpVt/Covexit?node-id=197686%3A4542
> Also, for icons we use react-icons. The ones you'll find on Figma are from feather-icons https://feathericons.com/

I created the component ButtonWithIcon where we can pass the label and icon, so we can use it in other places like menu and share buttons, where they also have their icons.

File: frontend/src/components/ButtonWithImage/ButtonWithImage.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import './ButtonWithImage.scss'
+
+function ButtonWithImage ({ label, icon, labelStyles, onClick }) {
+  return (
+    <div
+      onClick={onClick}
+      className="Button-with-image">
+      {icon}
+      <p
```
Comment (vellip) [inline frontend/src/components/ButtonWithImage/ButtonWithImage.js:10]: Let's not have this a paragraph. Semantically I think this is more like a div.

File: frontend/src/components/ButtonWithImage/ButtonWithImage.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import './ButtonWithImage.scss'
+
+function ButtonWithImage ({ label, icon, labelStyles, onClick }) {
+  return (
+    <div
```
Comment (vellip) [inline frontend/src/components/ButtonWithImage/ButtonWithImage.js:6]: Please use `<button>` for buttons. :D Or you integrate the <Button> component in here, which already does the semantic tag handling

**Pull request merged**